server/scripts/prisma_deploy_all/index.js

```diff
@@ -1,7 +1,7 @@
-const { env, run, queryManagement } = require('../helpers')
+const { env, deployPrismaService, deployAlgoliaIndex, queryManagement } = require('../helpers')
 
-const { PRISMA_URL } = env([
-  'PRISMA_URL',
+env([
+  'PRISMA_URL', // Implicitely required
```

Comment:
Oh. Well, `env` is used to retrieve variables from `process.env` but also, and most importantly, checking if they are present, and if not throws an error.

It's useful because if the variables are not present, it won't execute the script

Here, I don't need the variables in my file, but because I call `helpers` functions that need them, I still need to check if they are present.

`// Implicitely required` expresses that: We require this variable, but not directly in this file.

Reply:
Maybe helpers should take the vars as args then?

Reply:
Why not. I'll keep that in mind for the next time I modify the scripts